ui/app/pages/permissions-connect/index.scss

```
@@ -1,25 +1,36 @@
-@import 'permissions-connect-footer/index';
-
 @import 'choose-account/index';
 
 .permissions-connect {
   width: 100%;
   position: relative;
   background: white;
 
-  &__page-count-wrapper {
-    margin-top: 26px;
-    margin-left: 20px;
+  &__top-bar--two-items,
+  &__top-bar--right-item {
+    margin-top: 16px;
+    padding-right: 16px;
+    padding-left: 16px;
     display: flex;
-    justify-content: flex-end;
     align-items: flex-end;
+  }
+
+  &__top-bar--two-items{
```

Nit: missing space here
```suggestion
  &__top-bar--two-items {
```

---

Went away in cdd4c6125